code_examples/sdk_examples/src/core_features/messaging/index.ts

```diff
@@ -0,0 +1,72 @@
+import * as Kilt from '@kiltprotocol/sdk-js'
+import { createCompleteFullDid } from '../did/05_full_did_complete'
+import { createDriversLicenseCType } from '../claiming/01_create_ctype'
+import { decryptMessage } from './03_decrypt_message'
+import { encryptMessage } from './02_encrypt_message'
+import generateKeypairs from '../utils/generateKeypairs'
+import { generateMessage } from './01_generate_message'
+
+// Runs through the messaging encryption and decryption of messages
+export async function runAll(submitterAccount: Kilt.KiltKeyringPair) {
+  console.log('Running messaging flow...')
+  console.log('Generating a sender DID')
+  const sender = generateKeypairs()
+  const senderFullDid = await createCompleteFullDid(
+    submitterAccount,
+    {
+      ...sender
+    },
+    async ({ data }) => ({
+      signature: sender.authentication.sign(data),
+      keyType: sender.authentication.type
+    })
+  )
+  console.log('Generating a receiver DID')
+  const receiver = generateKeypairs()
+  const receiverFullDid = await createCompleteFullDid(
+    submitterAccount,
+    {
+      ...receiver
+    },
+    async ({ data }) => ({
+      signature: receiver.authentication.sign(data),
+      keyType: receiver.authentication.type
+    })
+  )
+
+  console.log(
+    'Generating a receiver credential for the encrypting and decrypting of the message'
```

Comment:
I know this is just for testing and internal use, but this message is not very clear.

Reply:
I have updated.